java/read-geopackage/README.md

```diff
@@ -1,25 +1,32 @@
 # Read GeoPackage
-Read rasters and feature tables from geopackages and display them as layers in a map.
 
-![Read GeoPackage App](read-geopackage.png)
+Add rasters and feature tables from a GeoPackage to a map.
+
+![Image of read GeoPackage](read-geopackage.png)
+
+## Use case
+
+The OGC GeoPackage specification defines an open standard for sharing raster and vector data. You may want to use GeoPackage files to support file-based sharing of geographic data.
 
 ## How to use the sample
-The `Layer`s in the `GeoPackage` are shown in a drawer on the left hand side of the screen. Click on a `Layer` to add or remove it from map.
+
+When the sample loads, the feature tables and rasters from the GeoPackage will be shown on the map.
 
 ## How it works
-1. Create a `GeoPackage` using a path a the local GeoPackage file.
-1. Load the `GeoPackage` with `loadAsync()`.
-1. Use `GeoPackage.getGeoPackageRasters()` to get each `GeoPackageRaster` and use them to make `RasterLayer`s.
-1. Use `GeoPackage.getGeoPackageFeatureTables()` to get each `GeoPackageFeatureTable` and use them to make `FeatureLayer`s.
-1. Based on user input, add or remove these `Layer`s from the `ArcGISMap`'s operationalLayers.
+
+1. Open the `GeoPackage` with a path and load it using `GeoPackage.loadAsync`.
+2. Iterate through available rasters, exposed by `geopackage.getGeoPackageRasters`.
```

Comment:
I think we are using class names over instance names?
```suggestion
2. Iterate through available rasters, exposed by `GeoPackage.getGeoPackageRasters`.
```

Reply:
I'll keep this as a variable name since the geopackage was "instantiated" above